uefi/src/result/status.rs

```diff
@@ -168,6 +168,59 @@ impl From<Status> for Result<(), ()> {
     }
 }
 
+impl core::fmt::Display for Status {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(
```

Comment:
This `Display` impl returns pretty much the same thing as the `Debug` impl, so you can simplify this `fmt` function to:
```rust
fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
    Debug::fmt(self, f)
}
```

Reply:
Ah, much better! Fixed.